src/main/resources/db/changelog/changes/04-create-users_role-table.yaml

```diff
@@ -0,0 +1,19 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-roles-to-user-table
+      author: Hadiuchko.Dmytro
+      changes:
+        - createTable:
+            tableName: users_roles
+            columns:
+              - column:
+                  name: user_id

```

Comment:
It's also good to make user_id + role_id a composite primary key, to disallow duplicate values.